docs/core/tools/dotnet-workload-search.md

```diff
@@ -0,0 +1,59 @@
+---
+title: dotnet workload search command
+description: The 'dotnet workload search' command searches for optional workloads.
+ms.date: 07/08/2021
+---
+# dotnet workload search
+
+**This article applies to:** ✔️ .NET 6 Preview SDK and later versions
+
+## Name
+
+`dotnet workload search` - Searches for optional workloads.
+
+## Synopsis
+
+```dotnetcli
+dotnet workload search [<SEARCH_STRING>]
+    [--sdk-version <VERSION>] [-v|--verbosity <LEVEL>]
+
+dotnet workload search -?|-h|--help
+```
+
+## Description
+
+The `dotnet workload search` command lists available workloads. You can filter the list by specifying all or part of the workload ID you're looking for.
+
+For more information about the `dotnet workload` commands, see the [dotnet workload install](dotnet-workload-install.md#description) command.
+
+## Arguments
+
+- **`SEARCH_STRING`**
+
+  The workload ID to search for, or part of it. For example, if you specify `tvos`, the command might list `microsoft-tvos-sdk-full` and `microsoft-net-runtime-tvos`.
```

Comment:
nit: these names will change with the newest rename. @jonathanpeppers are there more up to date names that we should use here?